src/amo/components/Collection/index.js

```diff
@@ -141,9 +141,12 @@ export class CollectionBase extends React.Component<Props> {
           <h1 className="Collection-title">
             {collection ? collection.name : <LoadingText />}
           </h1>
-          <p className="Collection-description">
-            {collection ? collection.description : <LoadingText />}
-          </p>
+          {collection ? (
+            <p
+              className="Collection-description"
+              dangerouslySetInnerHTML={sanitizeHTML(collection.description)}
```

Comment:
My understanding of this field's output is that shouldn't be any unescaped tags in it, so you should be able to tell the sanitizer that no tags are allowed. If you include HTML in the collection description entry, it should appear escaped in the API data already.

Reply:
`sanitizeHTML()` with only one argument means no tags are allowed, so that's what this is doing 😄